gnome/gingerblue/Portfile

```diff
@@ -57,5 +57,3 @@ depends_run         port:adwaita-icon-theme \
                     port:gstreamer1-gst-plugins-bad \
                     port:gstreamer1-gst-plugins-good \
                     port:gstreamer1-gst-plugins-ugly
-
-patchfiles          no-intltool-perl.patch
```

Comment:
This patch should not be removed. It is still relevant and will continue to be until the developers of intltool fix [the bug we reported to them in 2013](https://bugs.launchpad.net/intltool/+bug/1197875) (which you'll note is mentioned in the first line of the patchfile) and gingerblue regenerates its configure script with that fixed version of intltool. You can verify that this is the case by checking which version of perl intltool uses, installing the perl5 port with a different version of perl, and ensuring that the corresponding perl version of p5-xml-parser is not installed when configuring gingerblue. For example, today, intltool uses perl5.30, so I installed the perl5 port with the +perl5_32 variant and ensured that the p5.32-xml-parser port was not installed and configuring gingerblue generated the expected error, which the patch fixes:
```
checking for XML::Parser... configure: error: XML::Parser perl module is required for intltool
```
The patch does of course need to be regenerated since gingerblue 4.0.1's configure script was generated with a different version of autoconf than gingerblue 2.0.1's configure script.